build/changelog/build-changelog.php

```diff
@@ -4,9 +4,15 @@
 
 use Aws\Build\Changelog\ChangelogBuilder;
 
-$params = [
-    'verbose' => true
-];
+$params = array();
+
+$options = getopt('v:');
+
+if (isset($option['v'])) {
```

Comment:
Optional: a ternary `$params['verbose'] = isset($option['v']) ? $option['v'] : true` would be more concise.